apps/console/src/features/remote-repository-configuration/components/remote-fetch-details/remote-fetch-details-modal.tsx

```diff
@@ -0,0 +1,210 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import { AlertLevels, TestableComponentInterface } from "@wso2is/core/models";
+import { addAlert } from "@wso2is/core/store";
+import { CodeEditor, Hint, LinkButton, PrimaryButton, SegmentedAccordion } from "@wso2is/react-components";
+import { AxiosResponse } from "axios";
+import moment from "moment";
+import React, { FunctionComponent, ReactElement, SyntheticEvent, useEffect, useState } from "react";
+import { useDispatch } from "react-redux";
+import { Grid, Icon, Modal } from "semantic-ui-react";
+import { 
+    InterfaceConfigDetails, 
+    InterfaceRemoteConfigDetails, 
+    InterfaceRemoteFetchStatus, 
+    getConfigDeploymentDetails,
+    triggerConfigDeployment
+} from "../../../remote-repository-configuration";
+
+interface RemoteFetchDetailsProps extends TestableComponentInterface {
```

Comment:
Better to change `RemoteFetchDetailsProps ` -> `RemoteFetchDetailsPropsInterface`

Reply:
Fixed with commit 20f0488